modules/server/src/network/aggregations/AggregationAccumulator.ts

```diff
@@ -43,14 +44,14 @@ const resolveAggregations = ({
  * @param type
  * @param aggregations
  */
-export const resolveToNetworkAggregation = (
-	type: SupportedAggregation,
-	aggregations: Aggregations[],
-): AllAggregations | undefined => {
+const resolveToNetworkAggregation = (
+	type: string,
```

Comment:
This is no longer typed?

Reply:
Sorry not specific enough, no longer named type `SupportedAggregation` or an enum, just wildcard string?